contrib/backporting/common.sh

```diff
@@ -27,3 +27,10 @@ get_remote () {
   fi
   echo "$remote"
 }
+
+require_linux() {
+  if [ "$(uname)" != "Linux" ]; then
+      echo "$0: Linux required"
+      exit 1
+  fi
+}
```

Comment:
Nit: Looks like a missing newline here.

Do you know what limits those scripts to Linux only?

Reply:
Newline added.

I haven't investigated why the scripts only run on Linux, but I did observe that they didn't run on my macOS machine. The [backporting process documentation](https://docs.cilium.io/en/latest/contributing/release/backports/#one-time-setup) states:

> The scripts referred to below need to be run on Linux, they do not work on macOS.